azure-communication-ui/azure-communication-ui-demo-app/src/main/java/com/azure/android/communication/ui/callingcompositedemoapp/SettingsActivity.kt

```diff
@@ -0,0 +1,135 @@
+/**
```

Comment:
We can follow same pattern as other files

Reply:
Copyright message format changed.